apps/prairielearn/src/ee/routers/institutionAdmin.ts

```diff
@@ -15,17 +19,23 @@ const router = Router({ mergeParams: true });
 // We should change this in the future.
 router.use(authzIsAdministrator);
 
-router.use((req, res, next) => {
-  // The navbar relies on this property.
-  res.locals.urlPrefix = req.baseUrl;
-  next();
-});
+router.use(
+  asyncHandler(async (req, res, next) => {
+    // The navbar relies on this property.
+    res.locals.urlPrefix = req.baseUrl;
+
+    const hasLti13 = await features.enabled('lti13', { institution_id: req.params.institution_id });
+    res.locals.lti13_enabled = hasLti13 && isEnterprise();
```

Comment:
No need for an `isEnterprise()` check here - this router should only ever be attached to the server in an enterprise environment, so this is redundant.